src/docs/requirements.txt

```diff
@@ -1,5 +1,5 @@
 mkdocs>=1.2.3
-mkdocs-material>=8.0.1
+mkdocs-material>=8.0.2
```

Comment:
?

Reply:
this is from main branch

Reply:
ah odd; thought id merged main to the branch - fair enough!

Reply:
will remerge main